data/sql/2022_09_06_001_world_quest_9303_inoculación.sql

```diff
@@ -0,0 +1,32 @@
+-- Azuremist Isle: Inoculation
+SET @BUHO := 16518; -- Nestlewood Owlkin
+SET @INO  := 16534; -- Inoculated Nestlewood Owlkin
+
+-- Agregamos condicion para que ejecute el spell.
```

Comment:
@RaistlinM84 Please keep all the things related to the Pr in English, Including the name of the file, for other hand, you put the file in the wrong place, you have to put the files into the `Updates/pending` section into the corresponding DB, for example db_world

Reply:
if it was a rookie mistake, steve already told me about the placement of the file.